dallinger/recruiters.py

```diff
@@ -872,6 +903,42 @@ def external_submission_url(self):
         self.prolificservice.log_response(response)
         return response
 
+    def screen_out_allowed(self, participants: list[Participant]):
+        """
+        Check which participants in a list of participants can be screened out.
+
+        Returns a list of participants that can be screened out.
```

Comment:
I don't think that Prolific evaluates the list on a participant by participant basis, I think it just evaluates the whole list and then either screens out all of them or refuses to screen out any. i.e. the function should return bool rather than a list

Reply:
Changed to return a boolean now.